coderd/workspaces.go

```diff
@@ -349,7 +375,8 @@ func (api *API) postWorkspacesByOrganization(rw http.ResponseWriter, r *http.Req
 	dbTTL, err := validWorkspaceTTLMillis(createWorkspace.TTLMillis)
 	if err != nil {
 		httpapi.Write(rw, http.StatusBadRequest, httpapi.Response{
-			Message: "validate workspace ttl",
+			Message:  "Invalid workspace ttl",
```

Comment:
Do users know what tll is? If so, probably either capitalize it or spell it out. If not maybe we need a descriptive phrase.

Reply:
cc @ammario and @johnstcn

Reply:
Are we OK with `Time-To-Live (TTL)`?

Reply:
Is that term ever used in the UI? I can't tell whether it means time until it's live (time until auto-start) or time left to live (time until auto-stop) or something else.

Reply:
We use `Time until shutdown` on the form in the UI, which I feel is most clear